src/gffread/config.vsh.yaml

```
@@ -0,0 +1,396 @@
+        alternatives: -J
+        type: boolean_true
+        description: |
+          Discard any mRNAs that either lack initial START codon or the terminal STOP codon, or 
+          have an in-frame stop codon (i.e. only print mRNAs with a complete CDS).
+      - name: --no_pseudo
+        type: boolean_true
+        description: |
+          Filter out records matching the 'pseudo' keyword.
+      - name: --in_bed
+        type: boolean_true
+        description: |
+          Input should be parsed as BED format (automatic if the input filename ends with .bed*).
+      - name: --in_tlf
+        type: boolean_true
+        description: |
+          Input GFF-like one-line-per-transcript format without exon/CDS features (see --tlf option 
+          below); automatic if the input filename ends with .tlf).
+      - name: --stream
+        type: boolean_true
+        description: |
+          Fast processing of input GFF/BED transcripts as they are received (no sorting, exons must 
+          be grouped by transcript in the input data).
+
+  - name: Clustering
+    arguments:
+      - name: --merge
+        alternatives: -M
+        type: boolean_true
+        description: |
+          Cluster the input transcripts into loci, discarding "redundant" transcripts (those with 
+          the same exact introns and fully contained or equal boundaries).
+      - name: --dupinfo
+        alternatives: -d
+        type: file
+        description: |
+          For -M option, write duplication info to file <dupinfo>.
+      - name: --cluster_only
+        type: boolean_true
+        description: |
+          Same as -M/--merge but without discarding any of the "duplicate" transcripts, only create 
+          "locus" features.
+      - name: --rm_redundant
+        alternatives: -K
+        type: boolean_true
+        description: |
+          For -M option: also discard as redundant the shorter, fully contained transcripts (intron 
+          chains matching a part of the container).
+      - name: --no_boundary
+        alternatives: -Q
+        type: boolean_true
+        description: |
+          For -M option, no longer require boundary containment when assessing redundancy (can be 
+          combined with -K); only introns have to match for multi-exon transcripts, and >=80% overlap 
+          for single-exon transcripts.
+      - name: --no_overlap
+        alternatives: -Y
+        type: boolean_true
+        description: |
+          For -M option, enforce -Q but also discard overlapping single-exon transcripts, even on 
+          the opposite strand (can be combined with -K).
+
+resources:
+  - type: bash_script
+    path: script.sh
+test_resources:
+  - type: bash_script
+    path: test.sh
+  - type: file
+    path: test_data
+engines:
+- type: docker
+  image: quay.io/biocontainers/gffread:0.12.7--hdcf5f25_3
+  setup:
+    - type: docker
+      run: |
+        echo "gffread: \"0.12.7\"" > /var/software_versions.txt
```

Use `gffread --version` to get the version name. E.g.

```suggestion
        echo "gffread: \"$(gffread --version 2>&1)\"" > /var/software_versions.txt
```